configuration/sitemaps.md

```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
 
 As you can see, different Item data types are accepted as mappings values.
 The first two lines show very typical use cases.
-Imagine your TV as part of your openHAB setup.
-It's power state and channel number are internally represented by a binary Switch Item (OFF/ON) and a discrete number Item with only a few selectable states.
-By using a Switch or Selection element with a mappings array, you can replace these meaningless values by meaningful descriptions for your inputs in the user interface.
+Imagine your TV is part of your openHAB setup.
+Its power state is represented by a binary Switch Item.
+Its channel number is a discrete number Item that may only be set to one of three states.
+By using a Switch or Selection element with a mappings array, you can replace these meaningless values with user-friendly descriptions for display on the user interface.
+Using this mapping, the power state of the TV, instead of saying "ON" and "OFF" would more accurately display "on" and "standby".
```

Comment:
Add new line here?

"this mapping"?

Reply:
Adding a new line and rewording as below:
This mapping changes the displayed power state of the TV from "ON" and "OFF" to the more accurate terms, "on" and "standby".
Similarly, mapping above changes the numbers "1", "2", and "3" to "DasErste", "BBC One", and "Cartoon Network" respectively.

Reply:
This comment has been addressed.